src/components/AccordionTab/AccordionTab.tsx

```diff
@@ -1,37 +1,42 @@
+import { ReactNode } from 'react';
+
 import classNames from 'classnames';
 
-import Accordion from '../Accordion/Accordion';
+import Accordion from '~components/Accordion/Accordion';
+import Icon from '~components/Icon/Icon';
 
 import scss from './AccordionTab.module.scss';
 
 import { TAccordionTabProps } from './AccordionTab.types';
 
-function AccordionTab(props: TAccordionTabProps) {
-  const handleClose = () => {
-    if (props.onChangeOpen) props.onChangeOpen(!props.isOpen);
+function AccordionTab({
+  hideCloseButton = false,
+  isOpen = false,
+  ...props
+}: TAccordionTabProps): ReactNode {
+  const handleClose = (): void => {
+    if (props.onChangeOpen) props.onChangeOpen(!isOpen);
   };
 
-  const renderCloseButton = () => {
-    return (
-      <button onClick={handleClose} className={scss.closeButton}>
-        -
-      </button>
-    );
-  };
+  const renderCloseButton = (): ReactNode => (
+    <button className={scss.closeButton} onClick={handleClose} type="button">
```

Comment:
```suggestion
    <button className={scss.closeButton} onClick={handleClose>
```
esse `type="button"` não é necessário se você usa o html `button` ele só serve pra fazer atrocidades tipo usar div como button

Reply:
O pessoal estava aqui na call na hora que eu estava lendo e disseram que tem mais funcoes nisso, e disseram que e bom deixar.